tests/unit/Actions.spec.ts

```diff
@@ -113,7 +113,41 @@ describe('Actions Filing Functionality', () => {
         contactPoint: {
           email: 'someEmail',
           phone: '123-456-7890'
-        }
+        },
+        parties: [
+          {
+            person: {
+              id: 0,
+              firstName: 'Cameron',
+              lastName: 'Bowler',
+              middleName: 'D',
+              orgName: '',
+              partyType: 'Person'
```

Comment:
NB: in the unit tests, use a string like this instead of an enum type.
1. this is "data" that maybe the data provider didn't type, so no type here either
2. this makes the unit test more fragile, which is good for verifying that enum changes are tested

Reply:
Sounds good